plugins/mcreator-localization/help/default/particle/additional_expiry_condition.md

```diff
@@ -0,0 +1 @@
+If condition is specified, the particle must pass it in order to expire.
```

Comment:
Actually, the particle will expire at a set age, or earlier if condition passes. This is not additional requirement, but additional cancel option for particles.

Reply:
So, should I say it `might expire before its maximal age` if condition is passed?

Reply:
If a condition is specified, the particle can expire before its maximal age if the condition passes

Reply:
Nevermind,
:heavy_check_mark: Fixed